packages/angular/src/schematics/init/init.spec.ts

```diff
@@ -0,0 +1,546 @@
+        const folder = getFolder(isAngularCli, isWorkspace, 'memory');
+        expect(
+          targets?.['build']?.configurations?.['testronaut']
+        ).toMatchObject({
+          optimization: false,
+          extractLicenses: false,
+          sourceMap: true,
+          browser: `${folder}testronaut/main.ts`,
+          index: `${folder}testronaut/index.html`,
+          tsConfig: `${folder}testronaut/tsconfig.json`,
+        });
+
+        expect(targets?.['serve']?.configurations?.['testronaut']).toEqual({
+          buildTarget: 'memory:build:testronaut',
+          prebundle: false,
+        });
+
+        expect(infoLogger).toHaveBeenCalledWith(
+          'Testronaut successfully activated. Lift off!'
+        );
+        expect(errorLogger).toHaveBeenCalledTimes(0);
+      });
+
+      it('should add the testronaut files to the project', async () => {
+        const tree = await setup('test', isWorkspace);
+        ngAddGenerator(tree, { project: 'test' });
+        const folder = getFolder(isAngularCli, isWorkspace, 'test');
+
+        [
+          'main.ts',
+          'index.html',
+          'tsconfig.json',
+          '.gitignore',
+          'generated/index.ts',
+        ].forEach((file) => {
+          expect(
+            tree.exists(`${folder}testronaut/${file}`),
+            `File ${file} should exist in ${folder}`
+          ).toBe(true);
+        });
+
+        expect(tree.exists(`${folder}playwright-testronaut.config.mts`)).toBe(
+          true
+        );
+      });
+
+      it('should not add the examples by default', async () => {
+        const tree = await setup('test', isWorkspace);
+        ngAddGenerator(tree, { project: 'test' });
+        const folder = `${getFolder(
+          isAngularCli,
+          isWorkspace,
+          'test'
+        )}/src/testronaut-examples`;
+        expect(tree.exists(folder)).toBe(false);
+      });
+
+      it('shoud add examples when requested', async () => {
+        const tree = await setup('test', isWorkspace);
+        ngAddGenerator(tree, { project: 'test', createExamples: true });
+
+        const folder = `${getFolder(
+          isAngularCli,
+          isWorkspace,
+          'test'
+        )}src/testronaut-examples`;
+
+        expect(tree.exists(folder)).toBe(true);
+        expect(infoLogger).toHaveBeenCalledWith(
+          `Testronaut successfully activated for project test.${EOL}Study the examples in ${folder}.${EOL}Lift off!`
+        );
+      });
+
+      it("should start the test server by using the project's name", async () => {
+        const tree = await setup('maps', isWorkspace);
+        ngAddGenerator(tree, { project: 'maps', createExamples: true });
+
+        const configPath = `${getFolder(
+          isAngularCli,
+          isWorkspace,
+          'maps'
+        )}playwright-testronaut.config.mts`;
+
+        const config = tree.read(configPath, 'utf8') || '';
+
+        expect(config).toMatchSnapshot(
```

Comment:
I'd rather extract the section that interests us (e.g. command) even with a regex and make an explicit assertion here.
I think that it better expresses the intention and reduces the cognitive overhead of reviewing snapshots, especially when unrelated things change in the snapshots.